cmd/restic/args/args.go

```diff
@@ -0,0 +1,22 @@
+package args
```

Comment:
Can't `urfave/cli/v2` handle this?

Reply:
These are the arguments to pass to restic, not the args retrieved from the CLI. That's why I don't see `urface/cli/v2` for that purpose.

Reply:
You're more right than I was and it turns out this is just dead code.